xtask/src/main.rs

```diff
@@ -306,9 +310,11 @@ fn clean() -> Result<()> {
 }
 
 fn workspace() -> PathBuf {
-    Path::new(&env!("CARGO_MANIFEST_DIR"))
+    let mut cargopath = Path::new(&env!("CARGO_MANIFEST_DIR"))
         .ancestors()
         .nth(1)
         .unwrap()
-        .to_path_buf()
+        .to_path_buf();
+    cargopath.push("x86_64");
```

Comment:
Why are you pushing this?  This is no longer the workspace directory, but, rather a specific crate relative to the workspace root.